CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added - 2020-09-21
```

Comment:
As it has not been released yet, you don't need to add a date:

```suggestion
### Added
```

Reply:
Where is the button to accept and apply the maintainer's change? :D